standard/docs/en/implementation/hosting.md

```diff
@@ -21,85 +21,83 @@ However, very large files can be difficult for users to download and process. Th
+(a) Archiving a single-release release package for each release to a web accessible file system as it is created, and then regularly merging these releases to compile individual record files in the same file system. One approach might be to have a folder for each `ocid` and to place the releases and record related to that process into that folder. 
 
 (b) Providing access to releases and records through an API.
 
-Note that the second approach will require the API to maintain a full revision history of each contracting process, so that releases from each stage of a contracting process can be provided. 
+Note that the second approach needs the API to maintain a full revision history of each contracting process, so that releases from each stage of a contracting process can be provided. 
 
-Publishers should consider how to [ensure URIs can remain stable](https://www.w3.org/Provider/Style/URI.html), even across a change of systems.
+Publishers ought to consider how to [ensure URIs can remain stable](https://www.w3.org/Provider/Style/URI.html), even across a change of systems.
 
 ## Flattened serializations
 
 The [serialization](../../../serialization/) page provides details of how to generate 'flat' versions of OCDS data for use in spreadsheet software.
 
-The same principles discussed for bulk files above should be applied to CSV or Excel downloads of data.
+The same principles discussed for bulk files above ought to be applied to CSV or Excel downloads of data.
 
 
 ## Discovery and APIs
 
-There are many thousands of organizations who should be able to publish Open Contracting data. As a result, maintaining a central registry of all published data is impractical. Instead, OCDS proposes a common pattern for the discovery of Open Contracting data releases and records.
+There are many thousands of organizations who ought to be able to publish open contracting data. As a result, maintaining a central registry of all published data is impractical. Instead, OCDS proposes a common pattern for the discovery of releases and records.
 
 For the discovery of bulk datasets, and the location of any data feeds, we propose use of a data.json file.
 
-For the discovery of individual releases and records, we propose use atom feeds.
-
-There is a [community discussion here](https://github.com/open-contracting/standard/issues/290) on whether OCDS should include a proposed API standard in future iterations.
+For the discovery of individual releases and records, we propose use of Atom feeds.
 
 ### Dataset and feed discovery
 
-Publishers should provide a data.json document describing the location of all of the bulk OCDS files available for download. 
+Publishers can provide a data.json document describing the location of all of the bulk OCDS files available for download. 
```

Comment:
Is there a reason for the change from "should" to a non-normative synonym for "may"?

Reply:
The authors were very optimistic of the global adoption of the US standard. From the perspective of 2019, we can at most endorse that standard with a "can".

Reply:
Makes sense :-)